### PR TITLE
Add change log entry for changes between 0.3.0-ballot and the Aug balot version (https://github.com/hl7au/au-fhir-core/issues/141)

### LINKED ISSUE #141: Add change log page with entry for 0.3.0-ballot to Aug ballot
Now that we have a ballot for comment released that we took to Connectathon testing I suggest we have a change log page and track the significant changes between 0.3.0-ballot to the Aug ballot version.

This will should let the reader see how resolution of last ballot (and other changes) is incorporated coming into the next ballot.

### PR DESCRIPTION
**Pull request by dbojicic**
Fixes #141.

Changes made:
1. Enabled Change Log in the main menu
2. Added some placeholder text in the Change Log to indicate the kind of the content to be added soon

### CODE REVIEW

**Review by dt-r — CHANGES_REQUESTED** [review]
Remove the change tracking on the Home page. That is the purpose of the change log.

If this is merged first, Steph's PR has to be updated. If done the other way this PR needs to be updated.

**Comment by dbojicic** [comment]
@dt-r, PR updates, please review again.

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -7,4 +7,9 @@ This change log documents the significant updates and resolutions implemented fr
...
 #### Changes in this version
 
-TBD
\ No newline at end of file
+- Removed Must Support from the following elements in AU Core Procedure:
+  - Procedure.statusReason [FHIR-45013](https://jira.hl7.org/browse/FHIR-45013)
+  - Procedure.category [FHIR-45014](https://jira.hl7.org/browse/FHIR-45014)
+  - Procedure.recorder [FHIR-45015](https://jira.hl7.org/browse/FHIR-45015)
+  - Procedure.asserter [FHIR-45016](https://jira.hl7.org/browse/FHIR-45016)
+  - Procedure.bodySite [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017)
```
Comment (dt-r) [inline input/pagecontent/changes.md:15]: think this point on bodySite should also state that added profile specific implementation guidance on including body site information

**Review by dt-r — CHANGES_REQUESTED** [review]
one tweak

**Comment by dbojicic** [comment]
Updated, please review again.

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -12,4 +12,4 @@ This change log documents the significant updates and resolutions implemented fr
   - Procedure.category [FHIR-45014](https://jira.hl7.org/browse/FHIR-45014)
   - Procedure.recorder [FHIR-45015](https://jira.hl7.org/browse/FHIR-45015)
   - Procedure.asserter [FHIR-45016](https://jira.hl7.org/browse/FHIR-45016)
-  - Procedure.bodySite [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017)
+  - Procedure.bodySite [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017). Additionally, we've added profile-specific implementation guidance on including body site information.
```
Comment (dt-r) [inline input/pagecontent/changes.md:15]: The JIRA looks divorced from the inclusion of guidance. Make it:
`- Procedure.bodySite and added profile-specific implementation guidance on including bodysite information [FHIR-45017](https://jira.hl7.org/browse/FHIR-45017). `

**Review by dt-r — APPROVED** [review]

**Pull request merged**